### PR TITLE
Textual work report

### PR DESCRIPTION
**Pull request by njkim**
Add rest of the tabs for the textual work report.

### CODE REVIEW

File: afs/media/js/views/components/reports/textual-work.js [hunk 1]
```diff
@@ -37,38 +41,160 @@ define([
...
+
             self.summary = params.summary;
 
             if(params.report.cards){
                 const cards = params.report.cards;
                 
-                self.cards = self.createCardDictionary(cards)
+                self.cards = self.createCardDictionary(cards);
                 self.nameCards = {
                     name: self.cards?.['Textual Work Name'],
```
Comment (aarongundel) [inline afs/media/js/views/components/reports/textual-work.js:107]: noticed that the name cards are incorrect.

File: afs/media/js/views/components/reports/textual-work.js [hunk 2]
```diff
@@ -37,38 +41,160 @@ define([
...
                     type: self.cards?.['classification'],
-                }
-
+                };
                 self.descriptionCards = {
                     statement: self.cards.Statement,
                 };
-
                 self.existenceCards = {
                     'creation': {
```
Comment (aarongundel) [inline afs/media/js/views/components/reports/textual-work.js:116]: avoid putting json keys in quotes unless necessary - i.e., 'creation-time'

File: afs/media/js/views/components/reports/textual-work.js [hunk 3]
```diff
@@ -37,38 +41,160 @@ define([
...
+                            title: "Text Subjects", 
+                            data: [{
+                                key: 'subject(s) of textual work', 
+                                value: self.getRawNodeValue(self.resource(), 'subject'), 
+                                card: self.cards?.["subject(s) of textual work"],
+                                type: 'resource'
+                            }]
+                        },
+                        {
+                            title: "Textual Referent",
```
Comment (aarongundel) [inline afs/media/js/views/components/reports/textual-work.js:188]: Think you have a typo here - beyond that, should these be in one "aboutness" section?

File: afs/templates/views/components/reports/scenes/default.htm [hunk 4]
```diff
@@ -3,8 +3,13 @@
 {% block report %}
 
 {% block body %}
-<div class="afs-report-section" data-bind="foreach: {data: data().sections, as: section, noChildContext: true}">
-    <h2 class="afs-report-section-title"><i data-bind="click: function() {toggleVisibility(visible[section.title])}" class="fa fa-angle-double-right toggle"></i> <span data-bind="text: section.title"></span></h2>
+<div class="afs-report-section" data-bind="foreach: {data: data().sections, as: 'section', noChildContext: true}">
```
Comment (aarongundel) [inline afs/templates/views/components/reports/scenes/default.htm:6]: You'll want to do a pull here; I had a change to this sitting on my local branch that I had to push this morning, so. now there's a conflict.  Looks like you just expanded this out though, so it shouldn't be too difficult to resolve

**Review by aarongundel — CHANGES_REQUESTED** [review]
few minor changes

File: afs/media/js/views/components/reports/scenes/existence.js [hunk 5]
```diff
@@ -152,9 +167,11 @@ define(['underscore', 'knockout', 'arches', 'utils/report','bindings/datatable']
                         if(parts?.length){
                             var partObservables = parts.map(x => {
```
Comment (aarongundel) [inline afs/media/js/views/components/reports/scenes/existence.js:168]: Also noticed this var - use const or let.

File: afs/media/js/views/components/reports/textual-work.js (hunk 1 above)
Comment (njkim) [reply afs/media/js/views/components/reports/textual-work.js]: thanks. All the cards and nodes name are updated

File: afs/media/js/views/components/reports/textual-work.js (hunk 2 above)
Comment (njkim) [reply afs/media/js/views/components/reports/textual-work.js]: Fixed

File: afs/media/js/views/components/reports/textual-work.js (hunk 3 above)
Comment (njkim) [reply afs/media/js/views/components/reports/textual-work.js]: The titles have been updated with more meaningful names.
I believe these are from two separate cards.

File: afs/templates/views/components/reports/scenes/default.htm (hunk 4 above)
Comment (njkim) [reply afs/templates/views/components/reports/scenes/default.htm:6]: Merge conflict is resolved

File: afs/media/js/views/components/reports/scenes/existence.js (hunk 5 above)
Comment (njkim) [reply afs/media/js/views/components/reports/scenes/existence.js]: Fixed thanks.

**Pull request merged**